### DISCUSSION TITLE
java/collection/java-collection-questions-01

### DISCUSSION THREAD

**Discussion by giscus** [Snailclimb/JavaGuide#1659] [Announcements]
# java/collection/java-collection-questions-01

Java学习&&面试指南

https://javaguide.cn/java/collection/java-collection-questions-01.html

**Comment by svugithub** [comment]
how to learn java

**Comment by arjarya** [comment]
I am learning core java, can I get a job as a fresher on the base of only core java?

**Reply by Snailclimb** [reply]
> I get a job as a fresher on the base of only core java?

This may not be good, and you need to master some common frameworks and database usage...

**Reply by arjarya** [reply]
Thank You!

**Comment by Kiza1977Kiza** [comment]
> # java/collection/java-collection-questions-01
> Java学习&&面试指南
> 
> https://javaguide.cn/java/collection/java-collection-questions-01.html

**Comment by Kiza1977Kiza** [comment]
![com google android apps chromecast app](https://user-images.githubusercontent.com/119542201/204923766-07d35261-694f-46d7-b46f-34ea66b7acc1.png)

**Comment by Rams6990** [comment]
Hi if i only learn java can i get job in college placements?
